source/authentication/email-password.txt

```diff
@@ -115,17 +114,25 @@ confirmation email:
   parameters from the user's unique URL. Then, pass the ``token`` and ``tokenId``
   to the Client SDK's ``confirmUser`` function.
 
+  Mobile applications can handle email confirmation directly in the app. To do
+  this, configure `deep linking
+  <https://developer.android.com/training/app-links/deep-linking.html>`_ in
+  Android or `universal links
+  <https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html>`_
+  in iOS.
+
 - :guilabel:`Email Confirmation Subject`: the subject line for the email 
   that App Services sends. This value is optional if omitted, 
   App Services uses a default subject line instead. Custom email
   confirmation subjects can contain a maximum of 256 characters.
 
-.. note:: Use Deep Links in Mobile Apps
+.. note::
+
+   Confirmation emails are not currently customizable beyond the base URL and
+   subject line. In particular, they always come from a ``mongodb.com`` email
+   address. Please take this into consideration when deciding whether or not to
+   use email confirmation in your production app.
```

Comment:
It might be helpful to add something to this note about running a custom confirmation function if you need more customization. Just below this, the "Run a confirmation function" has a bullet that says:

> Send custom confirmation emails from a specific domain. Send these with a particular template using an external service.

It could be good to list some details like that in this note for folks who think they want email confirmation but want it to be more customized.